src/index.js

```diff
@@ -36,8 +36,15 @@ export default function loader(src) {
       src = Buffer.from(src);
     }
 
+    // Default to using base64 encoding unless the option was explicitly set to false.
+    const base64 = !(options.base64 === false);
+
+    const data = base64
+      ? src.toString('base64')
+      : escape(src.toString('binary'));
```

Comment:
Why `'binary'` (latin1) here? Wouldn't `'utf8'` be 'better'?

Reply:
`'binary'` is required for this to work with binary images (PNG, JPEG, etc)... which, while a bit unusual, seems to be perfectly valid.